internal/luasandbox/apis.go

```diff
@@ -114,3 +116,208 @@ func (l *LuaEnvironment) apiStrSplit() API {
 		`,
 	}
 }
+
+func (l *LuaEnvironment) apiGitReadBlob() API {
+	return &GoAPI{
+		Name:      "gitReadBlob",
+		Signature: "gitReadBlob(hash) -> blob bytes",
+		Help:      "Retrieve the Git blob in byte form from the repository with the provided hash.",
+		Examples: []string{
+			"gitReadBlob(abcdef) -> 'Hello, world!'",
```

Comment:
```suggestion
			"gitReadBlob("abcdef") -> 'Hello, world!'",
```

Reply:
can we use actual IDs here? we don't want to accidentally communicate support for shortened hashes and such...